Makefile

```diff
@@ -14,8 +21,19 @@ PLATFORM := $(ARCH)-$(OS)
 DIST_DIR := dist/$(PLATFORM)
 
 .PHONY: all clean mrproper build install uninstall test_install test \
-		benchmark debug bi coverage fast dist
-.SECONDARY: main-fast
+		benchmark debug bi coverage dist fast ci/fast.mk
+
+ifeq ($(MAKECMDGOALS), fast)
+-include ci/fast.mk
```

Comment:
hmm, i do not understand this one - this will include fast.mk and regenerate it in the next step?

Reply:
As I've checked, If you remove `ci/fast.mk` from .PHONY, you won't need to run `make fast` twice after doing `make clean`. Was there a particular reason why you put it there?